src/VisualStudio/CSharp/Impl/Options/Formatting/FormattingOptionPageControl.xaml.cs

```diff
@@ -52,5 +89,14 @@ private void SetNestedCheckboxesEnabled()
             FormatOnCloseBraceCheckBox.IsEnabled = FormatWhenTypingCheckBox.IsChecked == true;
             FormatOnSemicolonCheckBox.IsEnabled = FormatWhenTypingCheckBox.IsChecked == true;
         }
+
+        internal void SetCodeCleanupAsConfigured()
+        {
+            // IsCodeCleanupConfiguredCheckBox is hidden all the time, and it tracks if the user ever configured the code cleanup
```

Comment:
if it is not visible to user, why do you need checkbox? isn't Option sufficient?

Reply:
the option seems even roaming option (which is nice since if i ever saw it, I will never see it again)

Reply:
I don't have other ways to update the option value other than binding it with a checkbox....

Reply:
I'm not sure i understand.  Why isn't that just a normal Option we store... why does it have anything to do with the actual UI presentation layer?